metadata.json

```diff
@@ -68,7 +68,7 @@
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">=3.7.0 <5.0.0"
+      "version_requirement": ">=4.6.1 <5.0.0"
```

Comment:
shouldn't the `version_requirement` here now be `">=4.6.1 <6.0.0` instead of `5.0.0`?

Reply:
Yeah, I created this PR so long ago.

Reply:
I feel you. I've got a few of my own that have had to be rebased a handful of times.